include/pokeagb/battle/data.h

```diff
@@ -171,6 +171,23 @@ extern enum Item b_message_held_item;
  */
 POKEAGB_EXTERN u8 ball_number_to_ball_processing_index(u16 item_id);
 
+/**
+ * Contains the id of last used item
+ * @address{BPRE,0203AD30}
+ */
+extern enum Item b_last_used_item;
```

Comment:
Are you sure this is restricted to battles? I have this as `var_800E` in my IDB and it seems to be used for other non-battle item routes (e.g. escape rope). It would be better if you update the descriptions of the variables there.